ComposeSample/app/build.gradle

```diff
@@ -57,4 +58,7 @@ dependencies {
     testImplementation testDependencies.junit
     androidTestImplementation instrumentationTestDependencies.junit
     androidTestImplementation instrumentationTestDependencies.espressoCore
+}
+repositories {
+    mavenCentral()
```

Comment:
Do you need to add this repository? I don't see any extra dependency you have added in this PR.

If you need it, can you maybe add the `maven` repository to the group of repositories we have in the root `build.gradle` file, so we have all repositories there?  (https://github.com/microsoft/surface-duo-app-samples/blob/main/ComposeSample/build.gradle)

Reply:
Yes, you are right! We don't the `maven`. No idea why it was added automatically.